src/saveload/vehicle_sl.cpp

```diff
@@ -691,6 +691,7 @@ const SaveLoad *GetVehicleDescription(VehicleType vt)
 		 SLE_CONDVAR(Vehicle, profit_this_year,      SLE_INT64,                   SLV_65, SL_MAX_VERSION),
 		 SLE_CONDVAR(Vehicle, profit_last_year,      SLE_FILE_I32 | SLE_VAR_I64,   SL_MIN_VERSION,  SLV_65),
 		 SLE_CONDVAR(Vehicle, profit_last_year,      SLE_INT64,                   SLV_65, SL_MAX_VERSION),
+		 SLE_CONDVAR(Vehicle, profit_lifetime,       SLE_INT64,                  SLV_LIFETIME_PROFIT, SL_MAX_VERSION),
```

Comment:
Alignment looks slightly wrong here, seems to be missing a space before SLV_LIFETIME_PROFIT.

Reply:
Could fix profit_last_year a couple of lines above as well, while you're in the area